core/drand_daemon_control.go

```diff
@@ -155,9 +155,13 @@ func (dd *DrandDaemon) Shutdown(ctx context.Context, in *drand.ShutdownRequest)
 		if err != nil {
 			return nil, err
 		}
-		bp.Stop(ctx)
 
+		// TODO dlsniper: Check out
```

Comment:
This needs to be more explicit, I don't understand what you mean on that to-do

Reply:
Sorry, that was for me to do when I was looking at the code and I forgot it.